tests/extendrtests/tests/testthat/test-wrappers.R

```diff
@@ -1,3 +1,31 @@
+test_that("Wrapper code is up-to-date", {
+  # if this test fails check whether the wrapper code needs 
+  # updating via make_wrappers()
```

Comment:
I think there should be a 'comma' after 'fails', I tried reading this sentence several times before understanding what was meant.

Reply:
Completely rewritten this part.